docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-fdr-host-inventory.md

```diff
@@ -0,0 +1,149 @@
+To collect device data from the CrowdStrike platform, you must have an authorized CrowdStrike account. CrowdStrike APIs use an OAuth 2.0 authorization token to make authorized API calls. CrowdStrike API Client is required to get the OAuth 2.0 authorization token. To define a CrowdStrike API client, you must be designated as a `CrowdStrike Falcon Administrator` role.
+
+## Data sources
+
+CrowdStrike FDR Host Inventory retrieves device information every 24 hours. The source will fetch device information that was seen in the last 30 days for the first time. Afterward, it will only fetch information about devices that have been modified.
+
+## Setup and Configuration
+
+In this configuration, you will set up the CrowdStrike FDR account and configure it to be authorized and authenticated to use device information from CrowdStrike FDR API. To obtain the auth token, you will need the following parameters.
+
+### API Client and API Secret
+
+The **API security token** is used to authenticate with CrowdStrike FDR API. After successfully creating the API client, you will get the **Client Id**, **Client Secret**, and **Base URL**.
+To get the **CrowdStrike API Client**, follow the steps below:
+1. Access the [CrowdStrike FDR Platform](https://falcon.crowdstrike.com/login/)
+1. Enter your login credentials, including your email ID and password, to log in. You will be directed to the application dashboard once you've completed the two-factor authentication.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-login-screen.png')} alt="crowdstrike-fdr-host-inventory-login-screen.png" width="600" />
+1. From the CrowdStrike FDR Console, on the left-hand panel of the dashboard, locate the Menu option and click on it.<br/>
+<img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-open-menu.png')} alt="<crowdstrike-fdr-host-inventory-open-menu.png>" width="600" />
+1. Select the **Support and resources** option from the menu.
+<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-support-and-resources.png')} alt="<crowdstrike-fdr-host-inventory-support-and-resources.png>" width="600" />
+1. Navigate to **API clients and keys** and click on it. From there, you can view existing clients or
+add new API clients.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-api-key-client.png')} alt="<crowdstrike-fdr-host-inventory-api-key-client.png>" width="600" />
+1. Click **Add new API client**. You will be prompted to give a descriptive name and select the appropriate API scopes.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-new-client.png')} alt="<crowdstrike-fdr-host-inventory-new-client.png>" width="600" />
+1. Provide a proper name and description and select the **Read Hosts Scope**. Click on `ADD` to complete the process.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-add-client.png')} alt="<crowdstrike-fdr-host-inventory-add-client.png>" width="600" />
```

Comment:
Could you please mention any validations for the proper name? Users might think about how this proper name should look like.

Reply:
There is no validation for the name. Users can give any name which suits them.